userguide/tutorials/overdue.adoc

```diff
@@ -113,14 +130,17 @@ curl -v \
 "http://127.0.0.1:8080/1.0/kb/catalog/xml"
 ----
 
+In order to understand more about catalogs, you can refer to the https://docs.killbill.io/latest/userguide_subscription.html#components-catalog[subscription billing] document.
 
 === Overdue configuration
 
-The overdue configuration will define the three states the account will go through:
+Next, you need to define the overdue configuration XML file. Based on the scenario defined above, the overdue configuration will define the three states the account will go through:
+
+1. The account will enter the `WARNING` state 10 days after the first unpaid invoice (i.e. 20 days after the subscription creation)
```

Comment:
Might be worth restating that it is 10 days if trial -> first non $0 invoice + 10 days until we reach first overdue state

Reply:
Fixed